src/main/java/techreborn/tiles/generator/TileSolarPanel.java

```diff
@@ -44,61 +45,64 @@
 @RebornRegistry(modID = ModInfo.MOD_ID)
 public class TileSolarPanel extends TilePowerAcceptor implements IToolDrop {
 
-	@ConfigRegistry(config = "machines", category = "solar_panel", key = "SolarPanelMaxOutput", comment = "Solar Panel Max Output (Value in EU)")
-	public static int maxOutput = 32;
-	@ConfigRegistry(config = "machines", category = "solar_panel", key = "SolarPanelMaxEnergy", comment = "Solar Panel Max Energy (Value in EU)")
-	public static int maxEnergy = 1000;
-	@ConfigRegistry(config = "machines", category = "solar_panel", key = "SolarPanelEnergyPerTick", comment = "Solar Panel Energy Per Tick (Value in EU)")
-	public static int energyPerTick = 1;
+	//	@ConfigRegistry(config = "machines", category = "solar_panel", key = "SolarPanelMaxOutput", comment = "Solar Panel Max Output (Value in EU)")
+	//	public static int maxOutput = 32;
+	//	@ConfigRegistry(config = "machines", category = "solar_panel", key = "SolarPanelMaxEnergy", comment = "Solar Panel Max Energy (Value in EU)")
+	//	public static int maxEnergy = 1000;
+	//	@ConfigRegistry(config = "machines", category = "solar_panel", key = "SolarPanelEnergyPerTick", comment = "Solar Panel Energy Per Tick (Value in EU)")
+	//	public static int energyPerTick = 1;
 
 	boolean shouldMakePower = false;
 	boolean lastTickSate = false;
-
 	int powerToAdd;
+	private int generationRateD = 0;
```

Comment:
I would suggest putting the generation rates and other values in the enum

Reply:
They are, these are just placeholder values for the enum.

Reply:
You can just get them from the enum then? No need to have them here.

Reply:
Committed change, please confirm.